opm/autodiff/AquiferCarterTracy.hpp

```diff
@@ -294,7 +297,8 @@ namespace Opm
                 {
                     cellToConnectionIdx_[cell_idx_[idx]] = idx;
```

Comment:
I think there must be an error here: should not cellToConnectionIdx_ also be used by active cell (compressed) index, not cartesian index? Its size is at least equal to the compressed size.

Reply:
I think you are right. Fixed with the new commit.